week-4/Homework/mandatory/3-water-bottle.js

```diff
@@ -4,29 +4,36 @@ It will need a volume key to store how full or empty the bottle is.
 It will be 100 when full and 0 when empty. 
 Give your water bottle methods for filling it up, 
 drinking some of it, and emptying it.
-
 We made a start on this for you here: 
+
+
 */
 
+
 let bottle = {
   volume: 0,
   fill: function() {
+      this.volume = 100;
     // calling this function should make you bottles volume = 100;
   },
   drink: function() {
+      this.volume = -10;

```

Comment:
this does not decrease the volume, though. it's just going to assign the volume to -10. how would you solve this differently?